gitprompt.sh

```diff
@@ -490,6 +498,7 @@ function updatePrompt() {
   __GIT_PROMPT_IGNORE_STASH="${GIT_PROMPT_IGNORE_STASH:-0}"
```

Comment:
FWIW this feature is a little broken, here is a fix: 
```suggestion
  export __GIT_PROMPT_IGNORE_STASH="${GIT_PROMPT_IGNORE_STASH:-0}"
```